numpy/core/src/multiarray/flagsobject.c

```diff
@@ -807,11 +802,7 @@ NPY_NO_EXPORT PyTypeObject PyArrayFlags_Type = {
     0,                                          /* tp_print */
     0,                                          /* tp_getattr */
     0,                                          /* tp_setattr */
-#if defined(NPY_PY3K)
     0,                                          /* tp_reserved */
-#else
-    (cmpfunc)arrayflags_compare,                /* tp_compare */
```

Comment:
arrayflags_compare is still used by arrayflags_richcompare which is used on line 829

Reply:
Yeah, its OK to leave it for now, although it basically is simplified to a single `==`, so it is probably easier to remove it (it is too be useful as a helper).

Reply:
I'll add it to my list to simplify after.